storage-resize-images/functions/src/resize-image.ts

```diff
@@ -96,14 +103,14 @@ export const modifyImage = async ({
   contentType: string;
   size: string;
   objectMetadata: ObjectMetadata;
+  format: string;
 }): Promise<ResizedImageResult> => {
-  const { imageType } = config;
-  const hasImageTypeConfigSet = imageType !== "false";
-  const imageContentType = hasImageTypeConfigSet
-    ? supportedImageContentTypeMap[imageType]
+  const shouldFormatImage = format !== "false";
+  const imageContentType = shouldFormatImage
+    ? supportedImageContentTypeMap[format]
```

Comment:
Seems like we could end up in a situation where `imageContentType` is `undefined` if `format` doesn't exist in `supportedImageContentTypeMap`. Let's check if `format` is valid either here, and/or in `config.ts` and/or `convertType`.

(looks like this potential bug isn't new to this PR, but is probably worth fixing while we're here)

Reply:
Oh, `format` comes from a multiselect, not a free-form entry. Maybe not a big deal then? Might still help just to check in case we accidentally change the spelling on a format or something in the future